odoo/addons/base/static/src/css/contact_widget.css

```diff
@@ -0,0 +1,3 @@
+.o_portal_address [itemprop="telephone"] {
```

Comment:
can't we add it to `addons/portal/static/src/scss/portal.scss` that already contains CSS for `span[itemprop="name"]` and `div[itemprop="address"]`?

https://github.com/odoo/odoo/blob/fedd2fc6ba0eb1450dc38416bd0f0bd9161272ae/addons/portal/static/src/scss/portal.scss#L315-L320

Reply:
Won't this require the `portal` module to be installed? The styling issue comes from `base`.

Reply:
Adding the styling to `portal.css` doesn't work. Note that this issue only happens with PDF reports. PDF reports are generated in backend with wkhtmltopdf after the HTML report is generated from the template. Thus, wkhtmltopdf gets the styles from the links its finds in the template itself, not the styles shared with frontend like `portal.scss`.

Reply:
ok, then this seems very similar to https://github.com/odoo/odoo/pull/156778 for which ORM seemed ok with a similar fix

then I guess this could be ok here, but the current form doesn't seem great because we usually don't add CSS files for just a 3 lines widget and if we add a CSS file the normal way is to add it to a CSS file of the right asset (eg. here probably web.report_assets_common), there is two other ways that would be more ok to me:

- adding `style="white-space: nowrap;"` or class `text-nowrap` on the element
- adding the style in a CSS sheet that is used for printing report, eg /home/odoo/src/odoo/17.0/addons/web/static/src/webclient/actions/reports/report.scss but maybe there is a better file

Reply:
Adding to `report.scss` sounds good.